README.md

```diff
@@ -54,12 +80,18 @@ In this phase everyone is thinking like a Designer.  Design thinking also relies
 
 ***Q1.5 Based on the feedback, how might you change your solution?***
 
+Teleportation is too difficult, so we can just do the second VR solution. We don't want any impact on the game, which cameras strapped to the players' heads could do. Therefore, we are adding a new development: contact lenses with cameras in them, so we can still record from the players' points of view, but it won't affect how they play since it won't be noticeable.
```

Comment:
Great adjustment! I like this idea a lot since it can be used for the product you're catering to and for other purposes (i.e. recording live footage for players to review before their next games).